src/app/frontend/deploy/createnamespace_controller.js

```diff
@@ -67,10 +67,8 @@ export default class NamespaceDialogController {
     /** @export {!angular.FormController} */
     this.namespaceForm;
 
-    /**
-     * @export
-     */
-    this.i18n = i18n;
+    /** @export */
+    this.maxLength = 64;
```

Comment:
Agreed, particularly since the variable already exists a few lines up. I have removed this variable altogether in a new commit.